pointercrate-demonlist-pages/static/js/statsviewer/individual.js

```diff
@@ -161,7 +171,28 @@ $(window).on("load", function () {
     document.getElementById("statsviewer")
   );
 
-  window.statsViewer.initialize();
+  window.statsViewer.initialize().then(() => {
+    let url = window.location.href;
+    let params = new URLSearchParams(url.split('?')[1]);
+    let playerId = params.get('player');
+    if (playerId) {
+      window.statsViewer.selectPlayerByID(playerId)
+        .catch((err) => {
+          displayError(window.statsViewer)(err)
+          
+          // set the URL bar's value to the same location, but with the "player" parameter removed
+          params.delete("player");
+          const urlWithoutParam = `${window.location.origin}${window.location.pathname}?${params.toString()}`
+          window.history.replaceState({}, '', urlWithoutParam)
```

Comment:
Mh, why clear the player id from the navigation bar? If we're redirected to the statsviewer page from somewhere else, someone might just want to copy it. Maybe it'd be better to just change the URL to be the currently selected player every time a new player is selected?

Reply:
oh, sorry, I missed that part of it. Yeah, deleting it if the ID is bollocks makes sense, thanks for clarifying! (could I maybe bother you to add a code comment about that? Just in case I get confused again 4 years down the line lol)